augly/video/transforms.py

```diff
@@ -405,7 +461,12 @@ def apply_transform(
 
         @returns: the path to the augmented video
         """
-        return F.brightness(video_path, output_path, self.level, metadata=metadata)
+        return F.brightness(
+            video_path,
+            output_path,
+            level=self.chosen_value,
```

Comment:
The unit test workflow is currently failing because of this change, `self.chosen_value` is never defined -- please undo this & the other changes like this in this file!